kge/config-default.yaml

```diff
@@ -330,6 +330,23 @@ eval:
   # Type of evaluation (entity_ranking only at the moment)
   type: entity_ranking
 
+  entity_ranking:
+      # How to handle cases where the correct query answer has the same score as one or
+      # multiple wrong query answers. For example: Query: (s, p, ?). (Filtered) Answers:
+      # ['a': score 10, 'b': score 10, 'c': score 10, 'd': score 11, 'e': score 9].
+      # Correct query answer: 'a'. Possible options are:
+      # - rounded_mean_rank: Calculate the rounded mean rank. The final rank is
+      #                      calculated as 1 + #scores_higher +
+      #                      round_down(#answers_with_same_score / 2). In the example,
```

Comment:
We always round up to err on the worst_rank side.

Reply:
Ahh, I read this as "other_answers_with_same_score". Prehaps clarifiy: rounded (up) mean rank,

Reply:
I'm just describing what the code does.  And in ranks + num_ties // 2 ,  "//" rounds the rank down?

Reply:
See, you are confused as well. We round the number of ties (>=1) down, which corresponds to rounding the mean rank up. E.g., if there is **one** other answer with he same score than the correct one, then num_ties=**2**. We round down num_ties/2, but this corresponds to rounding up the mean rank. Perhaps its best to not show this computation here, it's only conusing. Mean rank rounded up is clear enough.